cucumber/authenticators_gce/features/authn_gce.feature

```diff
@@ -33,12 +33,41 @@ Feature: GCE Authenticator - Hosts can authenticate with GCE authenticator
     And I set "authn-gce/service-account-email" annotation to host "test-app"
     And I set "authn-gce/project-id" annotation to host "test-app"
     And I set "authn-gce/instance-name" annotation to host "test-app"
-    And I obtain a GCE identity token in "full" format with audience claim value: "conjur/cucumber/host/test-app"
+    And I obtain a GCE identity token in full format with audience claim value: "conjur/cucumber/host/test-app"
     And I save my place in the audit log file
```

Comment:
we need to remove audit, since we dont have audit in conjur and it may confuse.
` And I save my place in the log file`

Reply:
we don't have audit in conjur? doesn't it get written to the log?

if so, we need to remove this from all the tests that verify audit, not just here

Reply:
here it's ` And I save my place in the audit log file`
on you comment it's ` And I save my place in the log file` without `audit`

Reply:
@eladkug let's do this in another PR as it's out of scope of GCE. we have tests that verify the audit log but you say that there isn't one. 

i think that what happens here is that the tests assign a socket for audit and verifies that the audit is written there. this seems to be a good thing but we can challenge that. anyway, it should be outside of this PR imo

Reply:
is this the only blocker on this PR? @eladkug can we address this in another issue as it's not GCE realted?

Reply:
I agree it can be on different PR. @eranha please open a ticket for this

Reply:
The word `audit` is ignored in the step. I will create an issue to remove the word audit.
`# NOTE: The source code order of this step def and the one above matters.
And(/^The following appears in the (audit )?log after my savepoint:$/) do |_, message|
  expect(num_matches_since_savepoint(message)).to be >= 1
end`